po/common/common.pot

```diff
@@ -6069,4 +6069,8 @@ msgstr "The ingredients of the product must be specified in order to determine t
 
 msgctxt "nova_group_too_many_unknown_ingredient"
 msgid "We could not recognize some of the ingredients and determine the NOVA group."
-msgstr "We could not recognize some of the ingredients and determine the NOVA group."
\ No newline at end of file
+msgstr "We could not recognize some of the ingredients and determine the NOVA group."
+
+msgctxt "unselect_image"
+msgid "Unselect Image"
+msgstr ""
```

Comment:
@stephanegigandet to have the message at least in english if translation isn't available in target language,
 isn't it better to repeat msgid in msgstr, or is there an automatic fallback to en.po ?

Reply:
Product Opener does not use common.pot at all in fact, it reads only the *.po files. The common.pot is used by Crowdin.

In practice, we probably should have only one file, and configure crowdin to use en.po instead of common.pot
@teolemon is the master of the crowdin configuration

Reply:
This is related to GNU GetText.
per phrase.com:
POT, short for “portable object template”, is a format generated by GNU gettext to streamline the [software localization](https://phrase.com/blog/localization-and-translation-glossary/localization-l10n/) and [internationalization](https://phrase.com/blog/localization-and-translation-glossary/internationalization-i18n/). While source strings are placed after “msgid”, their translations are placed after “msgtr”. 

Note: POT files can sometimes be [PO files](https://help.phrase.com/help/gettext-po-files) as well. These two files are mostly identical except that PO files are generated by calling “msginit” in CMD. It is possible to perform the [translation](https://phrase.com/blog/localization-and-translation-glossary/translation-t9n/) on a POT file first and then rename it to a PO. The final localized format should be a machine-readable [“.mo” file](https://help.phrase.com/help/gettext-compiled-mo-files) generated from the PO file. The “.mo” file can be generated by calling “msgfmt” in CMD.

**In practise, Product Opener does not support .pot or .mo files**